docs/examples/WiFi/index.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: default
+title: "WiFi"
+date: 2021-10-10 12:17:00 --0600
+categories:
+---
+
+## Overview 
+
+Arista APs include an agent module (ocagent) that implements the OpenConfig interfaces. The agent module interacts with the rest of the AP modules for applying changes, fetching data, etc. For mor information on OpenConfig implementation in Arista WiFi, see [OpenConfig for Arista WiFi.](https://wifihelp.arista.com/post/openconfig-for-arista-wifi?term=openconfig&page=1)
+
+This page lists a few examples of the AP using gNMI.
+
+#### Get AP
+
+```shell
+python py_gnmicli.py -m 
+get -t <AP-IP> -x /provision-aps -user <username> -pass <password> -p <port> -g -o openconfig.mojonetworks.com
```

Comment:
is this script available somewhere, and/or is it possible to render this example using one of the common tools?